pyproject.toml

```diff
@@ -11,7 +11,7 @@ documentation = "https://docs.quiltdata.com"
 include = ["quilt3_local/catalog_bundle/*"]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.11"
+python = ">=3.7.1,<3.12"
```

Comment:
Can't we do this?
```suggestion
python = ">=3.7.1,<4"
```

Reply:
afair there were some issues with dependency resolution -- some dependencies did not support python3.11 and so refused to install